tests/app/test_party_service.py

```diff
@@ -22,83 +22,106 @@ def setUp(self):
-        mock_request.get(business_data_url, status_code=200, reason="OK", text='{"something": "else"}')
-        sut = PartyService()
+        ru_ids = ["1234", "5678"]
+        texts = '[{"something": "else"}, {"something": "else"}]'
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}&id={ru_ids[1]}"
+        mock_request.get(business_data_url, status_code=200, reason="OK", text=texts)
+        with self.app.app_context():
+            results = PartyService().get_business_details(ru_ids)
 
-        for retries in range(0, 2):
+        self.assertTrue(results, texts)
 
-            with self.app.app_context():
-                sut.get_business_details(ru)
+    @requests_mock.mock()
+    def test_multiple_user_uuids_hitting_party_service(self, mock_request):
+        """Test get business details sends a request and returns data"""
+        user_ids = ["c614e64e-d981-4eba-b016-d9822f09a4fb", "c614e64e-d981-4eba-b016-d9822f09a4f2"]
+        texts = '[{"something": "else"}, {"something": "else"}]'
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/respondents?id={user_ids[0]}&id={user_ids[1]}"
+        mock_request.get(business_data_url, status_code=200, reason="OK", text=texts)
+        with self.app.app_context():
+            results = PartyService().get_users_details(user_ids)
 
-        self.assertTrue(mock_request.call_count == 1)
+        self.assertTrue(results, texts)
 
     @requests_mock.mock()
-    def test_results_returned_from_get_business_get_executed_twice_for_different_ru(self, mock_request):
+    def test_multiple_business_uuid_one_invalid(self, mock_request):
+        """Test get business details sends a request and returns data"""
+        ru_ids = ["c614e64e-d981-4eba-b016-d9822f09a4fb", "12345"]
+        texts = "error"
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}&id={ru_ids[1]}"
+        mock_request.get(business_data_url, status_code=400, reason="Invalid uuid", text=texts)
+        with self.app.app_context():
+            results = PartyService().get_business_details(ru_ids)
+
+        self.assertEqual(results, [])
+
+    @requests_mock.mock()
+    def test_multiple_business_uuid_one_not_found(self, mock_request):
         """Test get business details sends a request and returns data"""
-        ru = "1234"
+        ru_ids = ["c614e64e-d981-4eba-b016-d9822f09a4fb", "c614e64e-d981-4eba-b016-d9822f09a4f2"]
+        business_data_url = f"{self.app.config['RAS_PARTY_SERVICE']}party-api/v1/businesses?id={ru_ids[0]}&id={ru_ids[1]}"
+        mock_request.get(business_data_url, status_code=200, text='{"something": "else"}')
+        with self.app.app_context():
+            results = PartyService().get_business_details(ru_ids)
 
-        sut = PartyService()
+        self.assertEqual(results, {"something": "else"})
```

Comment:
This isn't testing what it implies it is - if I added a uuid or removed one it would continue to pass.